tests/transformers/test_transformers_model_export.py

```diff
@@ -417,6 +417,27 @@ def feature_extraction_pipeline():
     return transformers.pipeline(model=model, tokenizer=tokenizer, task="feature-extraction")
 
 
+@pytest.fixture
```

Comment:
Can we mark the fixture with "module" scope" like `@pytest.fixture(scope="module")`

otherwise the fixture is executed per test running.

Reply:
In this case, I would not use a fixture because `snapshot_download` is only used once.

Reply:
pytest fixture is another complexity. If we don't need to use it, we don't need to.